htlcswitch/switch.go

```diff
@@ -150,6 +150,10 @@ type Config struct {
 	// the switch when a new block has arrived.
 	Notifier chainntnfs.ChainNotifier
 
+	// HtlcNotifier is an instance of a htlcNotifier which we will pipe HTLC
+	// events through.
+	HtlcNotifier htlcNotifier
```

Comment:
I'm actually a bit surprised that this works, that you can export a field of an unexported type 🤔